Assets/MixedRealityToolkit/_Core/Definitions/Devices/DeviceInputType.cs

```diff
@@ -10,61 +10,73 @@ namespace Microsoft.MixedReality.Toolkit.Internal.Definitions.Devices
     /// </summary>
     public enum DeviceInputType
     {
+        //Physical Input Type       -       Recommended Axis Type
         None = 0,
         Gaze,
         Voice,
         /// <summary>
         /// 6 Dof Pointer with position and rotation.
         /// </summary>
-        SpatialPointer,
+        SpatialPointer,                     // SixDof Axis
```

Comment:
Should we just move the recommended axis type to the summary?

Reply:
It's helpful documentation for now. Can remove in beta if need be.